### PR TITLE
1162 usergroup latest tech

### LINKED ISSUE #1162: User Group - LatestTech component
Current page: https://www.ssw.com.au/ssw/NETUG/Sydney.aspx
New design: https://www.figma.com/file/6IU1jg9Stosfecs4YfawoR/SSW-Website-v3?type=design&node-id=2612-8403&mode=dev

Description:
Need to develop a component for user-group pages to add a latest tech banner.

Image:
Image
**Figure: LatestTech banner on design**

### PR DESCRIPTION
**Pull request by johnxu16**
Fixes #1162 

Affected routes:

- NULL

Add done video, screenshots

image
**Figure: Demo on home page**

image
**Figure: Mobile demo on home page**

image
**Figure: Tina admin page**

image
**Figure: Edit badges collection**

image
**Figure: Badges collection**

image
**Figure: Badge config options**

**Figure: Control badge animation direction and duration (Angular in demo)**

### CODE REVIEW

**Comment by bradystroud** [comment]
@johnxu77 I can't get this working on the home page

**Comment by johnxu16** [comment]
> @johnxu77 I can't get this working on the home page


Oh, I deleted unused components in blocks-renderer.ts used by home page.

I can add it back for testing

@bradystroud 
It has been redeployed

**Review by bradystroud — CHANGES_REQUESTED** [review]
image

When there is no link, it should not be underlined

**Comment by bradystroud** [comment]
Hey @johnxu77 This looks really cool 🙂 

Can you film a done video on it?

**Comment by johnxu16** [comment]
> image
> When there is no link, it should not be underlined

Done

**Comment by johnxu16** [comment]
> Hey @johnxu77 This looks really cool 🙂
> 
> Can you film a done video on it?

I can 😊

File: components/blocks-renderer.tsx [hunk 1]
```diff
@@ -47,9 +45,7 @@ const componentMap = {
   LocationBlock,
   TableLayout,
   Agenda,
-  Organizer,
-  JoinGithub,
-  JoinAsPresenter,
```
Comment (bradystroud) [inline components/blocks-renderer.tsx:60]: Why are these being removed?

File: public/images/badges/Azure.png
Comment (bradystroud) [inline public/images/badges/Azure.png:1]: Dont we already have an Azure logo on the website?

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/blocks-renderer.tsx (hunk 1 above)
Comment (johnxu16) [reply components/blocks-renderer.tsx:60]: The page of 'User Group' may not use component `<Blocks />`

File: public/images/badges/Azure.png
Comment (johnxu16) [reply public/images/badges/Azure.png:1]: image
**Figure: difference between /logos/azure.png and /badges/Azure.png**

Comment (johnxu77) [reply public/images/badges/Azure.png:1]: image
**Figure: difference between /logos/azure.png and /badges/Azure.png**

**Comment by bradystroud** [comment]
Looks like there is a deployment issue @johnxu77 
Try fix it, ask @amankumarrr  if you need help

**Review by bradystroud — CHANGES_REQUESTED** [review]
Hey @johnxu77 

I noticed 2 issues

1. I think this field need a description - explain what it is and where you can create a config
2. This field is more of a heading, I would suggest making it a `string` instead of `rich-text`

image

**Comment by johnxu16** [comment]
> Hey @johnxu77
> 
> I noticed 2 issues
> 
> 1. I think this field need a description - explain what it is and where you can create a config
> 2. This field is more of a heading, I would suggest making it a `string` instead of `rich-text`
> 
> image

1. Done - description added

2. I use `rich-text` instead of `string` since it may has link on the text

File: components/blocks-renderer.tsx (hunk 1 above)
Comment (bradystroud) [reply components/blocks-renderer.tsx:60]: Ok, this is fine but if they are not going to be used as blocks, can we remove the schema from each component?

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Comment by johnxu16** [comment]
For now I would rather keep schema there, we used component not as a block but with schema as well. Ex. TestimoinalRow

Plus, those components need props including link and text to be set in use.

**Review by bradystroud — APPROVED** [review]

**Pull request merged**